app/packages/state/src/hooks/useKeyDown.tsx

```diff
@@ -0,0 +1,22 @@
+import { DependencyList, useEffect } from "react";
+
+/**
+ * a react hook that calls the handler when a given key is down
+ */
+export default function useKeyDown(
```

Comment:
In the future it might be nice to generalize native event hooks, so we don't have to enumerate each one 🤔 

Also, we should try to consolidate the `useEventHandler` hook in `hooks-utils.ts` so the pattern is clear. Maybe since these hooks aren't really stateful we create a new `@fiftyone/hooks` package

Reply:
+1 for `@fiftyone/hooks`, right now, they are all in state, so I added this one to state as well.